webpack-demo/webpack.config.js

```diff
@@ -9,7 +9,7 @@ module.exports = {
   },
   plugins: [
     new HtmlWebpackPlugin({
-      template: './src/index.html'
+      template: './src/index.html',
```

Comment:
- [ ] Good job with  your javascript code :+1: . But  on page load, the task array should be rendered dynamically to the DOM. The tasks should appear in order of the `index` values for each task.